cockroachcloud/architecture.md

```diff
@@ -0,0 +1,76 @@
+---
+title: CockroachCloud Architecture
+summary: Learn more about CockroachCloud architecture
+toc: true
+---
+
+CockroachCloud is a fully-managed deployment of CockroachDB. This page describes CockroachCloud's architecture and how it relates to CockroachDB.
+
+<div class="filters clearfix">
+  <button class="filter-button" data-scope="serverless">CockroachCloud Serverless </button>
+  <button class="filter-button" data-scope="dedicated">CockroachCloud Dedicated</button>
+</div>
+
+<section class="filter-content" markdown="1" data-scope="serverless">
+
+## CockroachCloud Serverless
+
+CockroachCloud Serverless (beta) is a fully-managed, multi-tenant deployment of CockroachDB. Being familiar with the following concepts will help you understand what our Serverless architecture achieves.
+
+### Concepts
+
+Term | Definition
+-----|-----------
+**Serverless cluster** | A cluster that’s automatically billed and scaled in response to the resources it consumes (as opposed to a dedicated cluster, which is billed and scaled statically).
+**Request Unit (RU)** | Request Units represent the compute and I/O resources used by a read or a write query. All database operations in CockroachCloud Serverless cost a certain amount of RUs depending on the resources used. For example, a "small read" might cost 1 RU, and a "large read" such as a full table scan with indexes could cost 100 RUs. You can see how many request Units your cluster has used on the Cluster Overview page.
+**Spend limit** | This is the maximum amount of money a user indicates that they would like to be billed in a particular billing period for a cluster. The actual amount a user is billed is based on the resources used during that billing period. A cluster's budget is allocated across storage, baseline performance, and burst performance.
+**Projected usage** | The amount of usage that we project a cluster will consume during a billing period. This is important for allocating a cluster’s spend limit, because we must leave enough budget to pay for storage for the rest of the billing period.
+**Baseline performance** | The minimum compute and IO performance that a user can expect from their cluster at all times. This is 100 RUs per second for all Serverless clusters (free and paid). The actual usage of a cluster may be lower than the baseline performance depending on application traffic, because not every application will need 100 RU/s at all times. 
```

Comment:
```suggestion
**Baseline performance** | The minimum compute and I/O performance that a user can expect from their cluster at all times. This is 100 RUs per second for all Serverless clusters (free and paid). The actual usage of a cluster may be lower than the baseline performance depending on application traffic, because not every application will need 100 RU/s at all times. 
```